src/main/java/com/miraisolutions/xlconnect/Workbook.java

```diff
@@ -417,10 +418,17 @@ public int[] getReferenceCoordinatesForTable(String sheetName, String tableName)
         return getReferenceCoordinatesForTable(workbook.getSheetIndex(sheetName), tableName);
     }
 
-    private void writeData(DataFrame data, Sheet sheet, int startRow, int startCol, boolean header) {
+    private void writeData(DataFrame data, Sheet sheet, int startRow, int startCol, boolean header, boolean overwriteFormulaCells) {
         // Get styles
         Map<String, CellStyle> styles = getStyles(data, sheet, startRow, startCol);
 
+        Consumer<Cell> formulaClear = overwriteFormulaCells ? (cell -> 
```

Comment:
Just a naming suggestion: maybe call it `clearFormula` to indicate the action of clearing. Or `maybeClearFormula` to indicate that it may or may not happen depending on the provided flag.